core/src/main/scala/org/apache/spark/sql/SnappyImplicits.scala

```diff
@@ -29,7 +29,7 @@ import org.apache.spark.{Partition, TaskContext}
  * Implicit conversions used by Snappy.
  */
 // scalastyle:off
-object snappy extends Serializable {
+object snappydata extends Serializable {
```

Comment:
Does this change require customer/user code base to be changed/recompiled in order to upgrade to 1.2 if they are using implicits like putInto?

Reply:
That's correct.
After discussion with Jags, Shirish and Nikhil we decided that let's change the Symbol name only for now instead of changing the name of the object which contains the implicits. Will close the PR.